torchaudio/functional_filtering.py

```diff
@@ -0,0 +1,160 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+import math
+import torch
+import math
+import torchaudio
+
+"""
+Biquad filtering approach based on SoX implementation 14.4.2
+
+Performance is currently very slow
```

Comment:
Have you seen performance gain by going FIR instead?

Reply:
and also compare to sox :)

Reply:
I will follow up with performance tests.  This was just a guess but I will test.